### PR TITLE
doc: Update bips.md for 0.19

### PR DESCRIPTION
**Pull request by laanwj**
- Add mention of BIP70 disabling by default at build time.

Any others?

E.g. does the burying of deployments of #16060 need to be mentioned? If so, where and how? For all of BIPs 34, 65 and 66?

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Could also link to the buried deployments pull request?

See
* ab0c35a290b8cb05b9bed0c24627bcc98bb22b2b
* e78aaf41f43d0e2ad78fa6d8dad61032c8ef73d0

**Comment by MarcoFalke** [comment]
ACK bcdab14673fd421fe3441980a0e50cce087420c2

**Comment by laanwj** [comment]
> Could also link to the buried deployments pull request?

Yes, but whether to put it under BIP9 or on the affected deployment BIPs?

**Comment by MarcoFalke** [comment]
It could be appended to each deployment, similar to how ab0c35a did it.

**Comment by laanwj** [comment]
I've decided to do both. Should be ready now.

**Comment by Sjors** [comment]
A few potential improvements:
* BIP 125 is used by the wallet, enabled by default in GUI as of v0.18.1 (PR #11605)
* [BIP 158](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki) Compact Block Filters for Light Clients can be indexed as of v0.19.0 (PR #14121)

File: doc/bips.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.18.0**):
 
-* [`BIP 9`](https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki): The changes allowing multiple soft-forks to be deployed in parallel have been implemented since **v0.12.1**  ([PR #7575](https://github.com/bitcoin/bitcoin/pull/7575))
+* [`BIP 9`](https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki): The changes allowing multiple soft-forks to be deployed in parallel have been implemented since **v0.12.1**  ([PR #7575](https://github.com/bitcoin/bitcoin/pull/7575)). The BIP 9 activation for BIPs 34, 65 and 66 was buried (hardcoded) in *v0.19.0* ([PR #16060](https://github.com/bitcoin/bitcoin/pull/16060)).
```
[the remark this answers is not in the document]
Comment (laanwj) [reply doc/bips.md]: OK, removing that then

File: doc/bips.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.18.0**):
...
-* [`BIP 34`](https://github.com/bitcoin/bips/blob/master/bip-0034.mediawiki): The rule that requires blocks to contain their height (number) in the coinbase input, and the introduction of version 2 blocks has been implemented since **v0.7.0**. The rule took effect for version 2 blocks as of *block 224413* (March 5th 2013), and version 1 blocks are no longer allowed since *block 227931* (March 25th 2013) ([PR #1526](https://github.com/bitcoin/bitcoin/pull/1526)).
+* [`BIP 34`](https://github.com/bitcoin/bips/blob/master/bip-0034.mediawiki): The rule that requires blocks to contain their height (number) in the coinbase input, and the introduction of version 2 blocks has been implemented since **v0.7.0**. The rule took effect for version 2 blocks as of *block 224413* (March 5th 2013), and version 1 blocks are no longer allowed since *block 227931* (March 25th 2013) ([PR #1526](https://github.com/bitcoin/bitcoin/pull/1526)). The BIP 9 activation for this softfork was buried (hardcoded) in *v0.19.0* ([PR #16060](https://github.com/bitcoin/bitcoin/pull/16060)).
```
[the remark this answers is not in the document]
Comment (laanwj) [reply doc/bips.md]: OK, will revert that change.

**Comment by laanwj** [comment]
I've removed the commit about the BIP 9 deployments, will leave writing that to someone else.

**Comment by MarcoFalke** [comment]
ACK 82c11773dc7493e1863b3b8277958f744a321eb9

**Review by hebasto — COMMENTED** [review]
Concept ACK.

In the first line
```
BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.18.0**):
```
s/18/19/

**Comment by MarcoFalke** [comment]
> s/18/19/

Might want to mention the version bump in the release process? https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#before-every-major-and-minor-release

**Comment by hebasto** [comment]
ACK 60e855f5c50c4578bbab93282226f236853491dc

**Pull request merged**